sources/Frontieres.cpp

```diff
@@ -601,6 +602,33 @@ void printParam()
                         myValue.c_str(), 100.0f);
             //            myValue = "Duration (ms): " + theCloud->getDurationMs();
             break;
+        case NUM:
+            myValue = _S("", "Cloud Num: ");
```

Comment:
la même chose en plus court sans `ostringstream`

```suggestion
            myValue = _S("", "Cloud Num: ") + std::to_string(theCloud.getId());
```

Reply:
en fait je me suis contenté de recopier la façon de faire qu'il y a sur tous les autres affichages de parametres, me semble t'il.
si on le change ici, ne faudrait il pas le changer partout ?

Reply:
ou alors estce parceque les autres parametres peuvent etre amenés a etre modifiés ?